### DISCUSSION TITLE
Pointing a data connection to an output file of a tool

### DISCUSSION THREAD

**Discussion by Mastomaki** [Spine-project/Spine-Toolbox#2515] [Support]
Hello!

The documentation says:

> Output files are files that will be archived into a timestamped result directory inside Tool’s data directory after the Tool specification has finished execution.

How do I point a data connection to an output file, if the file path changes every time (because of time stamp)? The file path in a data connection should not change.

**Comment by ptsavol** [comment]
What are you trying to achieve exactly? Data Connections are only useful as an input item. You can connect a Tool to a second Tool directly.

**Comment by Mastomaki** [comment]
Ok! It works that way. Then there is no problem.
